apps/api/src/app/subscribers-v2/usecases/remove-subscriber/remove-subscriber.usecase.ts

```diff
@@ -0,0 +1,82 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
```

Comment:
@desiprisg you can reuse the v1 usecase.

I suggest it moving to V2 and have the V1 controller (that will be deprecated) use the usecase from V2.